app/views/keywords/search.html.erb

```diff
@@ -0,0 +1,21 @@
+<div>
+  <div class="mb-2">
+    <p data-testid="search_result_search_term">Search term: <%= search_params[:search] %></p>
+    <p data-testid="search_result_query_type">Query type: <%= search_params[:query_type] %></p>
+  </div>
+
+  <p data-testid="search_result_total_matched_urls">Total matched URLs: <%= search_result_presenter.total_urls %></p>
+
+  <% search_result_presenter.search_result.each do |item| %>
+    <div data-keyword-id="<%= item[:keyword_id] %>">
+      <span data-testid="search_result_keyword_content">
+        Keyword content: <%= item[:keyword_content] %>
```

Comment:
just a suggestion, should we add the link to the `keywords/:id` to let the user to view that keyword details

Reply:
Fixed in 67326c06c9e333e39f71bc799260469d1f75f8dc